Demo/MozillaSocial-iOS/MoSoContent/Auth.swift

```diff
@@ -0,0 +1,138 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import AuthenticationServices
+
+let isProd = false
```

Comment:
nit: `isProd` could also be handled by an environment variable

Reply:
Similarly to a previous comment, keeping this contained will serve us later, otherwise you are completely correct.